src/Apps/__stories__/ArtworkApp.story.tsx

```diff
@@ -42,7 +42,7 @@ storiesOf("Apps/Artwork", module)
     return (
       <MockRouter
         routes={artworkRoutes}
-        initialRoute="/artwork/charles-arnoldi-brig-1"
+        initialRoute="/artwork/jeanine-coupe-ryding-ennui"
```

Comment:
I _think_ this artwork persists across staging syncs, so hopefully this storybook won't break as frequently

Reply:
Nice